### ISSUE TITLE
Issues with map rendering

### ISSUE THREAD

**Issue by devastator-pc** [naev/naev#731]
Problem best visible on attached screenshot - circles around systems in map view are sometimes less visible, probably due to interpolation issues. I tried various setting in video options, but nothing helps.

![map-render-issue](https://user-images.githubusercontent.com/30158703/28184127-5eb428c0-6812-11e7-9c60-a6615cf110fb.png)

Naev version: both 0.6.1 and 0.7.0
System: 64-bit Linux
Gnome 3.24 running on Wayland (instead of xorg)
Nvidia 1050 graphics card, Nvidia-provided drivers
Screen resolution:1920x1080

**Comment by devastator-pc** [comment]
I have just tried patching the source code, and it seems that using the following opengl options (instead of enabling them):
```
   glDisable(GL_LINE_SMOOTH);
   glDisable(GL_POINT_SMOOTH);
```
in the _map.c_ file, in method _map_renderSystems_ fixes this problem:

![map-render-issue-fixed](https://user-images.githubusercontent.com/30158703/28185085-4a1c3e62-6816-11e7-9b37-eb6c69663207.png)

I am just not sure if disabling smoothing is a proper fix for this issue. What is your opinion? Perhaps only do that on hi-res displays? (I did not encountered that problem on old 1366x768 display)
